backend/danswer/semantic_search/semantic_search.py

```diff
@@ -33,16 +35,6 @@
 
 logger = setup_logger()
 
-# Important considerations when choosing models
```

Comment:
these are just strings, looks so ugly here, surely nobody can say we didnt give credit for using some strings lul